modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -685,6 +909,86 @@ definition {
 		ObjectAdmin.assertObjectActionsNotPresent(actionName = "Action Name 2");
 	}
 
+	@description = "LPS-152181 - Verify that Action can be triggered after disabling the expression"
+	@priority = "4"
+	test CanTriggerActionAfterDisablingExpression {
+		task ("Given an object and a field is created") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object",
+				objectName = "CustomObject",
+				pluralLabelName = "Custom Objects");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "true",
+				objectName = "CustomObject");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject");
+		}
+
+		task ("Given an action is defined") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionDescription = "Custom Description",
+				actionName = "Custom Action Name",
+				active = "true",
+				inputExpression = "customObjectField == 'Entry Test'",
+				inputMethod = "true",
+				inputValue = "New object entry value",
+				objectLabel = "Custom Object",
+				thenAction = "Add an Object Entry",
+				whenAction = "On After Add");
+		}
+
+		task ("Given an entry with enabled condition") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customObjectField",
+				objectName = "CustomObject",
+				value = "Entry Test");
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject");
+
+			ObjectPortlet.viewEntry(entry = "New object entry value");
+
+			ObjectPortlet.deleteEntry(entry = "New object entry value");
+
+			ObjectPortlet.deleteEntry(entry = "Entry Test");
+		}
```

Comment:
We don't need to test the Action here since we're asserting that the Action can be triggered in other tests.